CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# dbt_servicenow v0.1.1
+[PR #7](https://github.com/fivetran/dbt_servicenow/pull/7) documents the DECISIONLOG in the README and moves the DECISIONLOG out of the /models folder, so that customers will be able to easily find it as a resource.
```

Comment:
Can you replicate the usual CHANGLOG format. For this you would make a claim that PR #7 includes the following changes and then provide a section for documentation updates?
```suggestion
[PR #7](https://github.com/fivetran/dbt_servicenow/pull/7) includes the following updates:

## Documentation Updates 
- Documents the DECISIONLOG in the README and moves the DECISIONLOG out of the `/models` folder, so that customers will be able to easily find it as a resource.
```

Reply:
oh yeah i can do that. i wasn't sure since this PR was more straightforward, but good call to keep it standard